.github/workflows/test.yml

```diff
@@ -10,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         cache_nonce: [ 1 ]
-        nim_version: [ 1.2.18, 1.4.8, 1.6.6 ]
+        nim_version: [ 1.6.6 ]
```

Comment:
Is 1.6.6 the latest nim we should be on?

Reply:
also, I think (I think!) if you just do `1.6` it should pick up the latest stuff.

Reply:
This workflow uses `choosenim`, which doesn't support the command `choosenim 1.6` as `1.6` isn't a version. `asdf` does support this command via `1.6:latest`, and one option is to use `asdf` if/when the `macos-13` (amd) runner is deprecated. Another option is `iffy`: https://github.com/codex-storage/asynctest/blob/main/.github/workflows/test.yml